tofu/geom/_core.py

```diff
@@ -4576,9 +4577,10 @@ def calc_signal(self, ff, t=None, ani=None, fkwdargs={}, Brightness=True,
         # Exclude Rays not seeing the plasma
         s = _GG.LOS_calc_signal(ff, Ds, us, res, DL,
                                 dmethod=resMode, method=method,
-                                t=t, Ani=ani, fkwdargs=fkwdargs, Test=True)
+                                t=t, ani=ani, fkwdargs=fkwdargs, Test=True)
```

Comment:
For the two changes above:
It seems to me more are and more obvious that we need to make a convention for naming variables. Mostly when to use capital letters or not. The branch Issue61 was not working because there were many instances where `DL` was being used instead of `Dl`, and in other functions we have the variable `DL` and `dL`. Keyword arguments also change from one function to the next one, eg. some times we have `func_a(..., test=True)` then `func_b(..., Test=True)`
Not only the difference in the capitalization gives no clue as to what is that variable, but it leads to confusion.
More explicit variable names would also imply less need for long comments/documentation.